configuration/raspi-config.md

```diff
@@ -56,42 +54,69 @@ Generally speaking, `raspi-config` aims to provide the functionality to make the
 <a name="menu-options"></a>
 ## Menu options
 
+### System Options
+
+The systems optiosn submenu allows you to make configuration changes to various parts of the boot, login and networking process, along with some other system level changes.
+
+#### Wireless LAN
+
+Allows setting of the wireless LAN SSID and passphrase.
+
+#### Audio
+
+Specifiy the audio output destination.
+
 <a name="change-user-password"></a>
-### Change User Password
+#### Password
 
 The default user on Raspberry Pi OS is ```pi``` with the password ```raspberry```. You can change that here. Read about other [users](../linux/usage/users.md).
  
-<a name="network-options"></a>
-### Network Options
-
-From this submenu you can set the host name, your wireless LAN SSID, and pre-shared key, or enable/disable predictable network interface names.
-
 <a name="hostname"></a>
 #### Hostname
 
 Set the visible name for this Pi on a network.
 
 <a name="boot-options"></a>
-### Boot Options
+#### Boot / Auto login
 
 From here you can change what happens when your Pi boots. Use this option to change your boot preference to command line or desktop. You can choose whether boot-up waits for the network to be available, and whether the Plymouth splash screen is displayed at boot-up.
```

Comment:
"choose whether boot-up waits for the network to be available" - that's controlled by the next option, not by this option.

"whether the Plymouth splash screen is displayed" - this option doesn't seem to be there?

The last sentence here should probably say something like "You can choose to have your Pi boot to a login prompt, or whether to log in as the `pi` user automatically."